esri2geojson/esri2geojson.py

```diff
@@ -25,18 +25,19 @@ def _collect_params(strings):
     
     return params
 
-def esri2geojson(esri_url, output_path, headers={}, params={}):
+def esri2geojson(esri_url, output_path, keep_sr, headers={}, params={}):
     ''' Convert single ESRI feature service URL to GeoJSON file.
     '''
     task = EsriRestDownloadTask('esri', params=params, headers=headers)
-    task.download(esri_url, output_path)
+    task.download(esri_url, output_path, keep_sr)
     
-parser = ArgumentParser(description='Convert single ESRI feature service URL to GeoJSON file.')
+parser = ArgumentParser(description='Convert single ESRI feature service URL to GeoJSON or CSV file.')
 
 parser.add_argument('esri_url', help='Required ESRI source URL.')
-parser.add_argument('geojson_path', help='Required output GeoJSON filename.')
+parser.add_argument('output_path', help='Required output filename.')
 
-parser.add_argument('-l', '--logfile', help='Optional log file name.')
+parser.add_argument('--keep_sr', help="Keep original projection",
```

Comment:
good point, we could make that option only available for the csv

On Tue, Aug 16, 2016 at 9:58 AM Ian Dees <notifications@github.com> wrote:

> In esri2geojson/esri2geojson.py
> <https://github.com/openaddresses/pyesridump/pull/3#discussion_r74952039>:
>
> >
> > -parser.add_argument('-l', '--logfile', help='Optional log file name.')
> > +parser.add_argument('--keep_sr', help="Keep original projection",
>
> *Technically* GeoJSON is required to be in epsg:4326
> <https://tools.ietf.org/html/rfc7946#section-4>. 😄
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openaddresses/pyesridump/pull/3/files/94ef6575f4615fa48eedaa2861763efe5de1d921..1c10dab1aa73d96880ecd85c24ed5e5638d0ba64#r74952039>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAgxbaK0NyEiTJ6rGqJCD_MPMVWM9JJIks5qgdARgaJpZM4Jj77m>
> .
>